src/App/App.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from './App.module.css';
+import { Notification } from '../components/notifications';
+import { IconError, IconWarning, IconSuccess  } from '../components/icons';
+
+export const App = () => {
+    return (
+        <div className={styles.container}>
+            <Notification color="var(--warning-color)" Icon={IconWarning} label="Notification" className={styles.notifBtn} />
+            <Notification color="var(--success-color)" Icon={IconSuccess} label="Notification" className={styles.notifBtn} />
+            <Notification color="var(--error-color)" Icon={IconError} label="Notification" className={styles.notifBtn} />
```

Comment:
```suggestion
         <Notification type="warning" className={styles.notifBtn}>Notification</Notifications>
        <Notification type="success" className={styles.notifBtn}>Notification</Notifications>
        <Notification type="error" className={styles.notifBtn}>Notification</Notifications>
```

Треба зробити так, щоб потім полегшити собі життя! Навіщо Icon і color, якщо можна передати `type` і вже в самому компоненті міняти кольори і іконки! 
також намагайся контент вставляти через children